### PR TITLE
fix: Artwork grid tablet layout

### PR DESCRIPTION
**Pull request by olerichter00**
This PR resolves [CX-2631]

### Description

This PR fixes a few artwork grid issues on tablets:

- Unifies the number of columns for tablets to **3** ([Slack thread with more context](https://artsy.slack.com/archives/C01B2P6LJUU/p1659436532361989).
- Adds correct right margin to the "Saved Works" artwork grid

| Saved Works | My Collection |
| --- | --- |
|Simulator Screen Shot - iPad Air (4th generation) - 2022-08-02 at 12 32 50 | Simulator Screen Shot - iPad Air (4th generation) - 2022-08-02 at 11 52 35|

### QA Test Case(s)

| Test Case | Feature | Environment | Acceptance Criteria | Setup Instructions/Link |
| --------- | :-----: | ----------: | ------------------: | ----------------------: |
|           |         |             |                     |                         |

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Fix artwork grids for tablets in landscape mode - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2631]: https://artsyproduct.atlassian.net/browse/CX-2631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/ArtworkGrids/GenericGrid.tsx [hunk 1]
```diff
@@ -50,11 +50,10 @@ export class GenericArtworksGrid extends React.Component<Props & PropsForArtwork
 
   layoutState(width: number): State {
     const isPad = width > 600
-    const isPadHorizontal = width > 900
 
-    const sectionCount = isPad ? (isPadHorizontal ? 4 : 3) : 2
+    const sectionCount = isPad ? 3 : 2
     const sectionMargins = this.props.sectionMargin ?? 0 * (sectionCount - 1)
-    const sectionDimension = (width - sectionMargins) / sectionCount
+    const sectionDimension = (width - sectionMargins - 20) / sectionCount
```
Comment (olerichter00) [inline src/app/Components/ArtworkGrids/GenericGrid.tsx:56]: Note: Make the 20 generic!

File: src/app/Components/ArtworkGrids/GenericGrid.tsx [hunk 2]
```diff
@@ -50,11 +50,11 @@ export class GenericArtworksGrid extends React.Component<Props & PropsForArtwork
...
   layoutState(width: number): State {
     const isPad = width > 600
-    const isPadHorizontal = width > 900
 
-    const sectionCount = isPad ? (isPadHorizontal ? 4 : 3) : 2
+    const sectionCount = isPad ? 3 : 2
     const sectionMargins = this.props.sectionMargin ?? 0 * (sectionCount - 1)
-    const sectionDimension = (width - sectionMargins) / sectionCount
+    const artworkPadding = 20
+    const sectionDimension = (width - sectionMargins - artworkPadding) / sectionCount
```
Comment (olerichter00) [inline src/app/Components/ArtworkGrids/GenericGrid.tsx:57]: `sectionDimension` was too large and the padding on the right side of the grid was missing.

File: src/app/Components/ArtworkGrids/GenericGrid.tsx [hunk 3]
```diff
@@ -50,11 +50,11 @@ export class GenericArtworksGrid extends React.Component<Props & PropsForArtwork
 
   layoutState(width: number): State {
     const isPad = width > 600
-    const isPadHorizontal = width > 900
 
-    const sectionCount = isPad ? (isPadHorizontal ? 4 : 3) : 2
+    const sectionCount = isPad ? 3 : 2
     const sectionMargins = this.props.sectionMargin ?? 0 * (sectionCount - 1)
-    const sectionDimension = (width - sectionMargins) / sectionCount
+    const artworkPadding = 20
```
Comment (MounirDhahri) [inline src/app/Components/ArtworkGrids/GenericGrid.tsx:56]: **nitpick:** 
```suggestion
    const artworkPadding = space(2)
```

**Review by MounirDhahri — APPROVED** [review]
niiice 🙌🙌🙌

**Pull request merged**